service/prometheus/service.go

```diff
@@ -115,16 +117,21 @@ func (s *Service) isReloadRateLimited() bool {
 	timeSinceLastReload := time.Since(s.lastReloadTime)
 
 	if timeSinceLastReload < s.minimumReloadTime {
-		s.logger.Log("debug", "ignoring reload request, only %t since last reload, minimum time between is %t", timeSinceLastReload, s.minimumReloadTime)
+		s.logger.Log("debug", fmt.Sprintf("ignoring reload request, only %s since last reload, minimum time between is %s", timeSinceLastReload, s.minimumReloadTime))
 		configurationReloadIgnoredCount.Inc()
 
 		return true
 	}
-	s.lastReloadTime = time.Now()
```

Comment:
That also looks like a bugfix to me. Setting the time when we reload makes way more sense to me than when asking if the reload is rate limited.

Reply:
Yeah, totally. Saw it while I was doing this PR - please excuse the inflation.